### ISSUE TITLE
Liquid Exception: Included file not found

### ISSUE THREAD

**Issue by charliegriefer** [jekyll/jekyll-help#77]
Hi all:

With the caveat that I'm not a Ruby person, and more of a back-end person in general... I'm trying to broaden my horizons and learn a bit.

I've set up a Jekyll site on Github Pages at http://charliegriefer.github.io. I'm using the [Minimal Mistakes theme](https://github.com/mmistakes/minimal-mistakes), which is awesome. Everything was going swimmingly until last night, when I wanted to add TravisCI (another new to me technology) to the mix.

Everything still builds and works fine locally. But my Travis build failed with the following:

> Liquid Exception: Included file '/home/travis/build/charliegriefer/charliegriefer.github.io/_includes/news_item.html' not found in vendor/bundle/ruby/1.9.1/gems/jekyll-1.5.1/site/news/index.html

I've "fixed" the issue by adding `news_item.html` to `_includes`, but I think we can all agree that this isn't really the proper fix :)

I've verified that Jekyll on my local machine (OS X Mavericks) is 1.5.1, which I believe is in use on Github Pages. My version of ruby shows as `ruby 2.0.0p451 (2014-02-24 revision 45167) [universal.x86_64-darwin13]`. I wouldn't think this would be an issue, even tho Github Pages uses 2.1.1, but it's the only difference that I can see.

In a nutshell, the Minimal Mistakes theme already had a Gemfile and gemfile.lock. I modified those as part of getting Travis CI set up. I'm probably not intelligent enough (yet) to be modifying Gemfiles. But I'd imagine that maybe something there went terribly wrong (in spite of the fact that my jekyll build and serve work fine locally).

I don't see _anywhere_ in my files where I'd be calling `news_item`, and it's driving me nuts.

Any thoughts/suggestions/advice would be greatly appreciated. If there's any additional information that I should be providing, please let me know.

Thanks!
Charlie
